test-cases/manager/manager-backup-and-restore-5TB-single-node.yaml

```diff
@@ -0,0 +1,28 @@
+test_duration: 4320
+
+prepare_write_cmd: ["cassandra-stress write cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..1342177280",
+                    "cassandra-stress write cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1342177281..2684354560",
+                    "cassandra-stress write cl=ONE n=1342177280 -schema 'replication(factor=1) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=2684354561..4026531840",
```

Comment:
replace all cl=ONE with cl=ALL.
Now it "doesn't matter" but if one later decide to change the test to more than one node, it will matter and it's better one won't fall in this trap.